### PR TITLE
test(developer): keyboard info compiler messages unit tests 2

### PR DESCRIPTION
**Pull request by markcsinclair**
Some tidying up of earlier unit tests, mainly introducing some constants.

Continues earlier PR [#10848](https://github.com/keymanapp/keyman/pull/10848)

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts [hunk 1]
```diff
@@ -234,9 +186,9 @@ describe('KeyboardInfoCompilerMessages', function () {
...
 
   it('should generate ERROR_LicenseIsNotValid error if license file is invalid', async function() {
-    const jsFilename = makePathToFixture('khmer_angkor', 'build', 'khmer_angkor.js');
-    const kpsFilename = makePathToFixture('khmer_angkor', 'source', 'khmer_angkor.kps');
-    const kmpFilename = makePathToFixture('khmer_angkor', 'build', 'khmer_angkor.kmp');
+    const jsFilename = KHMER_ANGKOR_JS;
+    const kpsFilename = KHMER_ANGKOR_KPS;
+    const kmpFilename = KHMER_ANGKOR_KMP;
 
     const sources = {
```
Comment (mcdurdin) [inline developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts:193]: You could do this as:

```js
const sources = {
  ...KHMER_ANGKOR_SOURCES,
  sourcePath: 'release/k/invalid-license',
};
```

here, and above, and that way eliminate a bunch of repeated constants. :grin:

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts (hunk 1 above)
Comment (markcsinclair) [reply developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts:184]: I found only a couple of places I could do this; can you spot any more?

Comment (mcdurdin) [reply developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts:184]: didn't spot any more!

**Review by mcdurdin — APPROVED** [review]
LGTM, nice and clean

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.314-beta](https://keyman.com/downloads/releases/beta/17.0.314)

**Pull request merged**